packages/manager/apps/web/client/app/domain/index.js

```diff
@@ -0,0 +1,15 @@
+import generalInformationsState from './general-informations/domain-general-informations.state';
+import webhosting from './webhosting';
+
+import routing from './domain.routing';
+
+const moduleName = 'ovhManagerWebDomainModule';
+
+angular
+  .module(moduleName, [
+    webhosting,
+  ])
+  .config(routing)
+  .config(generalInformationsState);
```

Comment:
Seems like you have two state declaration for your module, could it be splitted ?

Reply:
In fact, the better is to create a module that contains `general-informations`code.

But this implies refactor of the two controllers `domain.controller.js` and `domain-general-informations.controller.js`. And this is a huge task since they share values through the `$scope`.